app/models/comment.rb

```diff
@@ -1,2 +1,12 @@
 class Comment < ApplicationRecord
+  belongs_to :user
+  belongs_to :post
+
+  after_save :update_counter
+
+  private
+
+  def update_counter
+    user.increment!(:PostCounter)
+  end
```

Comment:
- Please in reading the project requirements, it stated that "Comments has a method that updates the **comments counter** for a **post**".  Kindly check the code above and correct the mistake.

Reply:
Hi @OLIPLICHE,
Thanks for your feedback.
The change that you required it's already solved.